app/controllers/purchases_controller.rb

```diff
@@ -40,6 +40,6 @@ def category
   private
 
   def purchase_params
-    params.require(:purchase).permit(:date, :description, :amount, :payee, :subcategory, :user)
+    params.require(:purchase).permit(:date, :description, :amount, :payee, :subcategory)
```

Comment:
Should this also include `:purchaser` and `:purchasee`?

Reply:
I believe those cases are covered/caught by the 'belongs_to :purchaser ...' lines in the model, so adding them at this point would be redundant - what's the best practice?